Week2/mehmet.js

```diff
@@ -0,0 +1,167 @@
+// 1. Write a console.log statement saying "Hello World!" for each language that you know.
+
+console.log(`Merhaba Dünya!
+Hello World!
+Ciao Mondo!
+Hallo Wereld!`);
+
+// 2. Consider the following code:
+
+console.log(`I'm awesome`);
+
+// 3. Declare a variable x and initialize it with an integer, using these exact steps:
+// 3.1
+let x; // Declaration and initialization
+
+// 3.2
+console.log(`the value of my variable x will be: undefined`);
+
+// 3.3
+console.log(x);
+
+// 3.4
+x = 9;
+
+// 3.5
+console.log(`the value of my x is 9`);
+
+// 3.6
+console.log(x);
+
+// 4
+let y = `house number 80`;
+
+// 4.1
+console.log(`my string's \(y\) value will be \"house number 80\"`);
+
+// 4.2
+console.log(y);
+
+// 4.3
+y = `house number 90`;
+
+// 4.4
+console.log(`my new string \(y\) value will be \"house number 90\"`);
+
+// 4.5.
+console.log(y);
+
+// 5. How do you round the number 7.25, to the nearest integer (i.e., whole number)?
+
+// 5.1
+const z = `7.25`;
+
+// 5.2
+console.log(z);
+
+// 5.3
+const a = Math.round(z);
+
+// 5.4
+console.log(a);
+
+// 5.5
+const highest = Math.max(z, a);
```

Comment:
> 👍 for using `const` here instead of `let`. I would also like you to explore the difference between `Math.floor` and `Math.round` methods 🙂

I read the explanations and differences, thanks!